packages/core/saved-objects/core-saved-objects-migration-server-internal/src/actions/calculate_exclude_filters.test.ts

```diff
@@ -98,5 +99,6 @@ describe('calculateExcludeFilters', () => {
     expect((result as Either.Right<any>).right.errorsByType.type1.toString()).toMatchInlineSnapshot(
       `"Error: excludeFromUpgrade hook timed out after 0.1 seconds."`
     );
+    timer1?.unref();
```

Comment:
unrelated to other changes, but this was causing jest to complain about open handles